daml-foundations/daml-ghc/src/DA/Service/Daml/Compiler/Impl/Dar.hs

```diff
@@ -54,11 +53,9 @@ buildDar dalf modRoot dalfDependencies fileDependencies name = do
     -- Reads all module source files, and pairs paths (with changed prefix)
     -- with contents as BS. The path must be within the module root path, and
     -- is modified to have prefix <name> instead of the original root path.
-    absModRoot <- makeAbsolute modRoot
```

Comment:
Did anything change to make this unnecessary or did we just forget to change this when we moved to relative paths?

Reply:
I think it was just not necessary.